jsk_arc2017_baxter/euslisp/lib/util.l

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env roseus
+
+(require "package://jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/util.l")
+
+(ros::load-ros-manifest "jsk_gui_msgs")
+(ros::load-ros-manifest "jsk_recognition_msgs")
+
+(defun opposite-arm (arm) (if (eq arm :larm) :rarm :larm))
+
+(defun get-bin-contents (bin)
+  (ros::get-param (format nil "/bin_contents/~A"
+                            (string-upcase (symbol2str bin)))))
+
+(defun wait-for-user-input-to-start (arm)
```

Comment:
+1 to @knorth55 
This is util function and depends on usage of two arms in picking by baxter.